fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/PreparedStatementHelper.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.database.utils.common;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.util.Calendar;
+
+/**
+ * Collection of utility functions to simply setting values on a PreparedStatement
+ */
+public class PreparedStatementHelper {
+    // The PreparedStatement we delegate everything to
+    private final Calendar UTC = CalendarHelper.getCalendarForUTC();
+    private final PreparedStatement ps;
```

Comment:
better if the comment is associated with the variable i think
```suggestion
    private final Calendar UTC = CalendarHelper.getCalendarForUTC();
    // The PreparedStatement we delegate everything to    
    private final PreparedStatement ps;
```